python/gtsam/examples/CustomFactorExample.py

```diff
@@ -0,0 +1,154 @@
+import gtsam
+import numpy as np
+
+from typing import List, Optional
+from functools import partial
+
+# Simulate a car for one second
```

Comment:
Make a function for this that you call below.

Reply:
done